StepDefinitions/HotelsSD/HotelAddGusSD.js

```diff
@@ -0,0 +1,53 @@
+// const { Then, Given } = require("@wdio/cucumber-framework");
+// const { expect } = require("chai");
+// const HotelHomePage = require("../../Pages/Hotels/HotelHomePage");
+
+// const hotelhomepage = new HotelHomePage();
+
+// Given(/^I am on hotels homepage$/, async function() {
+//     await browser.url('/');
+//     await browser.pause(1000);
+// })
+
+// Then(/^I click on Travelers$/, async function() {
+//     await hotelhomepage.clickTravelersBox();
+//     await browser.pause(2000);
+// })
+
+// Then(/^I select “Adults" as 6$/, async function() {
+//     await hotelhomepage.add6Adults();
```

Comment:
You can pass this num from feature file as parameter like below

Then(/^I select “Adults" as (.+)$/, async function(num) {
//     await hotelhomepage.add6Adults(num);